packages/web-forms/src/components/widgets/TextMedia.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import ImageBlock from '@/components/widgets/ImageBlock.vue';
+import type { TextRange } from '@getodk/xforms-engine';
+import { computed } from 'vue';
+
+interface TextMediaProps {
+	readonly label: TextRange<'item-label'>;
+}
+
+const props = defineProps<TextMediaProps>();
+
+const text = computed(() => props.label.asString);
+const image = computed(() => props.label.imageSource);
+const video = computed(() => props.label.videoSource);
+const audio = computed(() => props.label.audioSource);
+</script>
+
+<template>
+	<span v-if="text != null" class="text-content">{{ label.asString }}</span>
```

Comment:
`text` instead of `label.asString` for consistency?

Reply:
I like that!

Reply:
Done